tests/unit/dataactbroker/test_account_handler.py

```diff
@@ -99,4 +103,115 @@ def test_max_login_failure(monkeypatch):
                     "Please contact DATABroker@fiscal.treasury.gov to obtain access."
 
     # Not in cgac group
-    assert error_message == json.loads(json_response.get_data().decode("utf-8"))['message']
\ No newline at end of file
+    assert error_message == json.loads(json_response.get_data().decode("utf-8"))['message']
+
+
+def test_list_submissions_success(database, job_constants, monkeypatch):
+    ah = dataactbroker.handlers.accountHandler.AccountHandler(Mock(), InterfaceHolder())
+
+    mock_value = Mock()
+    mock_value.getName.return_value = 1
+    monkeypatch.setattr(dataactbroker.handlers.accountHandler, 'LoginSession', mock_value)
+
+    user = UserFactory(user_id=1, cgac_code='cgac')
+    sub = SubmissionFactory(user_id=1, submission_id=1, number_of_warnings=1, cgac_code='cgac')
+    add_models(database, [user, sub])
+
+    json_response = ah.list_submissions(None, None, None)
+    assert json.loads(json_response.get_data().decode("utf-8"))['total'] == 1
+    assert json.loads(json_response.get_data().decode("utf-8"))['submissions'][0]['status'] == "validation_successful_warnings"
+    delete_models(database, [user, sub])
+
+    sess = database.session
+    user = UserFactory(user_id=1, cgac_code='cgac')
+    sub = SubmissionFactory(user_id=1, submission_id=1, cgac_code='cgac')
+    job = JobFactory(submission_id=1, job_status=sess.query(JobStatus).filter_by(name='finished').one(),
+                     job_type=sess.query(JobType).filter_by(name='csv_record_validation').one(),
+                     file_type=sess.query(FileType).filter_by(name='award').one())
```

Comment:
Can we use the dictionaries in `dataactcore.models.lookups` rather than querying the db? Something like:
```python
job = JobFactory(submission_id=1, job_status_id=JOB_STATUS_DICT['finished'], ...
```

Also, are the `job_type` and `file_type` relevant to this test? If not, we can just use the randomly-generated values to make this test easier to read.